analyzer/codechecker_analyzer/analyzers/cppcheck/analyzer.py

```diff
@@ -83,6 +83,29 @@ def get_analyzer_mentioned_files(self, output):
         """
         pass
 
+    def parse_analyzer_config(self):
+        "Parses a white listed compiler flags"
+        params = []
+        interesting_option = re.compile("^-[I|U|D].*$")
```

Comment:
Is this the set? Can we do something more formal, like for each interesting option, something like this:

```python
# The above extend() won't properly insert the analyzer_option
# in case of the following format -I <path/to/include>.
# The below check will add the next item in the
# analyzer_options list if the parameter is specified with a
# space, as that should be actual path to the include.
interesting_options.append("-I.*");
# The bloopity blooper needs special handling.
interesting_options.append("-bLoOoOp.*");
```

Reply:
Make a judgement call on this, I don't insist if its very incovenient.

Reply:
With the added explanation, I think it will suffice.